pkgs/by-name/ka/katawa-shoujo-re-engineered/package.nix

```diff
@@ -0,0 +1,67 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  makeDesktopItem,
+  copyDesktopItems,
+  makeWrapper,
+  renpy,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "katawa-shoujo-re-engineered";
+  version = "1.4.4";
+
+  src = fetchFromGitHub {
+    /*
+      # alternative source:
```

Comment:
what is the official one?

Reply:
The codeberg one is the main.

![image](https://github.com/NixOS/nixpkgs/assets/23431373/de368b3f-e2c9-4faf-918e-9b3b6bc5c081)

Reply:
Not at the current time.

`srcs` is usually employed when the source spreads over multiple places (e.g. engine + assets), but not for specifying backup mirrors in this fashion...

P.S.: given the recent XZ Incident, use the official one.

Reply:
Not at the current time.

`srcs` is usually employed when the source spreads over multiple places (e.g. engine + assets), but not for specifying backup mirrors in this fashion...

P.S.: given the recent XZ Incident, use the official one.